Sources/SQLKit/Exports.swift

```diff
@@ -1,3 +1,3 @@
-@_documentation(visibility: internal) @_exported import protocol NIO.EventLoop
-@_documentation(visibility: internal) @_exported import class NIO.EventLoopFuture
```

Comment:
Note, there is a way to perform this elision for non-wasm builds only, if desired. This seemed like the best solution, though, since it appears that only NIOCore is being used currently in this repository.

Reply:
@gwynne That is a perfectly valid concern. I pushed a change that narrows preserves the old export and SPM dependency behavior, and makes the change only for WASI platforms. WASI cannot compile with the full `NIO.EventLoop` qualifier. It will only compile with `NIOCore.EventLoop`. So it wouldn't be possible for the latest changeset to break downstream WASI platform compilation, as they would be broken as-is without these changes.

And with the additional guards, non-WASI downstream dependencies should be 100% unaffected by this change, other than the bump in the minor NIO version.

Is there anything else you would like me to look into for this PR?

Reply:
@gwynne From what I see, all CI checks including the new wasm check are passing on this MR now.